ideascube/library/static/library/library.css

```diff
@@ -39,4 +39,145 @@ img.cover {
 .book-detail h3 {
     font-family: 'fira_sansmedium';
     font-size: 1em;
-}
\ No newline at end of file
+}
+
+
+
+/* ************************************************* */
+/* ************ SECTION COLORS ********************* */
+/* ************************************************* */
+
+/************* Cartoons *************/
+#id_section > option[value=children-cartoons] {
```

Comment:
To avoid leaving this thread hanging, I have verified that this was done in the latest push to this branch. (and it looks beautiful)